packages/ui-components/src/Badge/Badge.module.scss

```diff
@@ -0,0 +1,45 @@
+@import "../styles/tokens.scss";
+
+.badge {
+  border-radius: 3px;
+  display: inline-block;
+  font-size: 12px;
+  line-height: 14px;
+  padding: crl-gutters(0.25) crl-gutters(0.75);
+  vertical-align: crl-gutters(0.5);
+  text-align: center;
+  white-space: nowrap;
+  width: fit-content;
+}
+
+.intent--neutral {
```

Comment:
@nathanstilwell , what do you think about following BEM naming conventions?

Reply:
I am choosing not to use BEM here as I'm trying to represent a different paradigm (classnames based on props rather than classnames based on DOM structure and state). I am going to change the format slightly (see [comment](https://github.com/cockroachdb/ui/pull/4#issuecomment-589183018)) to better align with [Astroturf conventions](https://github.com/4Catalyzer/astroturf#component-api) (changing from `${propName}--${propValue}` to `${propName}-${propValue}`).